### PR TITLE
[Extractor] Re-added command line arguments

### LINKED ISSUE #319: Please bring back command line arguments to the extractor
With the latest update it appears you have to use extractorparams.json to define your arguments to the extractor tool. This prevents scenarios in which you need to pass dynamic values as arguments to the tool.

### PR DESCRIPTION
**Pull request by bremnes**
Should fix #319.

### CODE REVIEW

**Comment by bremnes** [comment]
@RupengLiu, I've done some small adjustments to the work you did when introducing `extractorConfig`. Are you okay with extractorparams.json no longer being default used, and that you have to specify `--extractorConfig path/to/file` if you want to use the file alternative? I think it gives the users more alternatives.

That means that by running just `extract` it will fail with "Error occured: Missing parameter <sourceApimName>." because neither file nor command line arguments are provided.

**Comment by bremnes** [comment]
I noticed that the arguments to `extract` is starting to become messy. There are some options that could be boolean, and there are other that are duplicates. We can deal with that in another issue or pull request.

File: src/APIM_ARMTemplate/apimtemplate/apimtemplate.csproj [hunk 1]
```diff
@@ -8,11 +8,8 @@
   </PropertyGroup>
 
   <ItemGroup>
```
Comment (bremnes) [inline src/APIM_ARMTemplate/apimtemplate/apimtemplate.csproj]: I'm not sure why these folders were removed in the first place, but Visual Studio automatically removed the models folder when I re-introduced it and moved `ExtractorConfiguration.cs` there.

**Comment by RupengLiu** [comment]
> @RupengLiu, I've done some small adjustments to the work you did when introducing `extractorConfig`. Are you okay with extractorparams.json no longer being default used, and that you have to specify `--extractorConfig path/to/file` if you want to use the file alternative? I think it gives the users more alternatives.
> 
> That means that by running just `extract` it will fail with "Error occured: Missing parameter ." because neither file nor command line arguments are provided.

Hi @bremnes , I'm ok with this.

**Comment by RupengLiu** [comment]
Hi, @bremnes Nice work!

**Pull request merged**